resources/css/app.css

```
@@ -16,23 +16,25 @@
 /* ----------------------------------------------------------
-   Redefine Flux's `zinc` scale → semantic surface tokens.
+   Redefine Flux's `zinc` scale → Flexoki base gray scale.
    Flux uses `zinc` for nearly every gray (sidebar, borders,
-   text, inputs). Pointing each stop at our paper/ink/ui
-   tokens re-skins the entire surface system in one shot.
+   text, inputs). The base scale is static (does NOT flip
+   between modes), so the canonical `bg-zinc-50 dark:bg-zinc-900`
+   pattern keeps working: light bg in light mode, dark bg in
+   dark mode. Visual identity stays Flexoki (warm gray family).
    ---------------------------------------------------------- */
 @theme {
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)